tensorflow/tensorflow.bzl

```diff
@@ -2013,12 +2013,18 @@ def cc_header_only_library(name, deps = [], includes = [], extra_deps = [], comp
         **kwargs
     )
 
-def tf_custom_op_library_additional_deps():
+def tf_custom_op_library_additional_deps_internal():
     return [
         "@com_google_protobuf//:protobuf_headers",  # copybara:comment
         clean_dep("//third_party/eigen3"),
         clean_dep("//tensorflow/core:framework_headers_lib"),
-    ] + if_windows([clean_dep("//tensorflow/python:pywrap_tensorflow_import_lib")])
+    ]
+
+def tf_custom_op_library_additional_deps():
```

Comment:
Is there any other place that still use this? I have this question because I wonder why don't you just fix this definition and not adding X_internal.

Reply:
I think it is used in `tf_custom_op_library` macro (user-defined custom ops). There is also a Windows-specific custom logic with `tf_custom_op_library_additional_deps_impl` related to library exports. I looked into [PR](https://github.com/tensorflow/tensorflow/pull/16659) that added this, but couldn't fully understand why it was needed. So I was just afraid to break anything and thought that this was the least invasive solution.

Reply:
It looks to me that tf_custom_op_library_additional_deps_impl is not used anywhere.
tf_custom_op_library_additional_deps is used by tf_custom_op_library, but shouldn't we replace that with the new tf_custom_op_library_additional_deps_internal also? If not, why not?

Reply:
I see.

The sole reason why I changed it for TF-TRT is because it was forming a dependency cycle: TF-TRT -> pywrap_tensorflow_import_lib -> TF-TRT. Currently they are linked together anyway, so I removed pywrap_tensorflow_import_lib from TF-TRT dependencies. But `tf_custom_op_library` is not connected to TF-TRT so I wanted to leave it untouched to not break anything.

`tf_custom_op_library_additional_deps_impl ` is used [here](https://github.com/tensorflow/tensorflow/blob/r2.8/tensorflow/BUILD#L1070-L1079). However, it does not seem to call `tf_custom_op_library_additional_deps` so I think it will not break if I change `tf_custom_op_library_additional_deps`.

How about this: instead of creating `X_internal`, I just move `pywrap_tensorflow_import_lib` inside `tf_custom_op_library`? In theory, nothing will break this way, and the circular dependency will be resolved, too.

Reply:
Updated with the method I wrote above. I also followed [this tutorial](https://www.tensorflow.org/guide/create_op#compile_the_op_using_bazel_tensorflow_source_installation) and confirmed that everything works as intended